t/lib/t/MusicBrainz/Server/Data/ArtistCredit.pm

```diff
@@ -7,6 +7,7 @@ use Test::Moose;
 use Test::Fatal;
 use Test::More;
 
+use MusicBrainz::Script::RemoveUnreferencedRows;
```

Comment:
This works for now, but it probably makes sense to move the test under t/script/, since it's really testing the script now and not anything in the data layer.

Reply:
Well, it's also testing that it got added to the table to begin with, but I can move it to script in any case.